### PR TITLE
added "experiment" type method to pdb_search

### PR DESCRIPTION
**Pull request by tomshani**
I added the necessary addition to the unit test...hope there's no issue

### CODE REVIEW

**Review by jbenjoseph — CHANGES_REQUESTED** [review]
Can you run this code though the black formatter?

`pip install pre-commit && pre-commit install && pre-commit run --all-files`

File: aiondata/raw/protein_structure.py [hunk 1]
```diff
@@ -244,50 +254,54 @@ def fetch_PDB_uniprot_accession(pdb_id):
...
                 return None
-        
+
         except requests.RequestException as e:
             print(f"Request failed: {e}")
             return None
 
-
-
     def fetch_uniprot_sequence(uniprot_id):
```
Comment (jbenjoseph) [inline aiondata/raw/protein_structure.py]: Missing self parameter

**Review by jbenjoseph — CHANGES_REQUESTED** [review]
Make sure self is the first parameter of all Object methods.

File: aiondata/raw/protein_structure.py (hunk 1 above)
Comment (tomshani) [reply aiondata/raw/protein_structure.py]: fixed

**Pull request merged**